internal/exp/metrics/identity/scope.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package identity
+
+import (
+	"hash"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+)
+
+type Scope struct {
+	res Resource
+
+	name    string
```

Comment:
Do we need to include the scope schemaURL as part of the identifier?

Reply:
Leave it out IMO